src/main/java/bio/terra/common/PdaoConstant.java

```diff
@@ -14,4 +11,7 @@ private PdaoConstant() {
     public static final String STAGING_TABLE_ROW_ID_COLUMN = PDAO_PREFIX + STAGING_TABLE_PREFIX + "row_id";
     public static final String TARGET_TABLE_ROW_ID_COLUMN = PDAO_PREFIX + TARGET_TABLE_PREFIX + "row_id";
 
+    public static String prefixName(String name) {
```

Comment:
Not very constant. Maybe rename this module to PdaoNames.

Reply:
Was only needed when splitting out the new file, will move it back to the PDAO